tests/test_test_utils.py

```diff
@@ -334,9 +335,32 @@ def test_noop(self) -> None:
     result.stdout.fnmatch_lines(["*RuntimeError*"])
 
 
-async def test_server_context_manager(app, loop) -> None:
-    async with TestServer(app, loop=loop) as server:
-        async with aiohttp.ClientSession(loop=loop) as client:
```

Comment:
Backports must not modify unrelated code. This should be reverted to its 3.11 state.

Reply:
Ok, I will do the same